node-fs-writefile/cp.js

```diff
@@ -0,0 +1,20 @@
+import { readFile, writeFile } from 'node:fs/promises';
+
+const fileToCopy = process.argv[2];
+const newFile = process.argv[3];
+
+async function read(file) {
+  const contents = await readFile(file, 'utf8');
+  return contents;
```

Comment:
You could eliminate the `contents` local variable with:
`return await readFile(file, 'utf8');`